packages/expo-image-picker/ios/Errors.swift

```diff
@@ -0,0 +1,110 @@
+// Copyright 2022-present 650 Industries. All rights reserved.
+
+import ExpoModulesCore
+
+internal struct PermissionsModuleNotFoundError: CodedError {
```

Comment:
Could you migrate these errors to be the exceptions?
```suggestion
internal class PermissionsModuleNotFoundException: Exception {
```